pom.xml

```diff
@@ -268,6 +268,13 @@
                     <excludes>
                         <exclude>**/*KT.java</exclude>
                     </excludes>
+                    <!-- Added to fix
+                        https://github.com/testcontainers/testcontainers-java/issues/892
+                        https://github.com/testcontainers/testcontainers-java/issues/700#issuecomment-457541087
+                    -->
+                    <environmentVariables>
+                        <TESTCONTAINERS_RYUK_DISABLED>true</TESTCONTAINERS_RYUK_DISABLED>
```

Comment:
As I commented before, this was the approach to make it work before upgrading to **docker-ce** but the creator of the library ( the same of the answer you refer to ) said : 
```Jonathan Vila  
Hi, I have an issue with RYUK. My tests are running ok if I set an environment var TESTCONTAINERS_RYUK_DISABLED=true in the run configuration in IntelliJ.... but I need this to run ok if I do mvn install ..... Is there a way to set this var globally and programmatically to TestContainers ?

Sergei Egorov 
No, there is no way to set it from the code, and you really should not
I would recommend checking why it fails on a first place
Ryuk is a critical part of Testcontainers and without it you most probably will have dangling containers

Jonathan Vila
ohhh, ok

Sergei Egorov  
we introduced the environment variable for CIs that perform the cleanup themselves (like BitBucket) and forbid accessing the Docker socket```

Reply:
Thanks for reporting the conversation.
I agree with him: we should check why it fails.
I edited my https://github.com/project-xavier/xavier-integration/pull/20#issuecomment-515354000 to have him adding his thoughts.

Reply:
With the **docker-ce** version it works fine.
This is the conversation I had yesterday afternoon with him : 
```Sergei Egorov 14:53
The thing is… the one shipped with Fedora (just “docker”) seems to be pretty old (January 2017) and I assume lower than the version we support
What you can try is to lower the Docker client requirement of https://github.com/testcontainers/moby-ryuk
in case you have any experience with Golang
(if I remember correctly, the only issue was that Ryuk fails to start)
perhaps due to the requested API version
Jonathan Vila 15:32
nop I dont have experience with Golang
Jonathan Vila 17:50
Apparently everything solved:
Sergei Egorov 17:56
yay 
Jonathan Vila 17:56
https://github.com/openshift/origin/issues/20581

```

Reply:
In the end the API version doesn't seem to be the issue since we're fine with their minimum requirement that, as far as i understand, is Docker 1.6.0 from https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/DockerClientFactory.java#L167
It seems to be all about security and running Ryuk in privileged mode that it's something they have a property for https://www.testcontainers.org/features/configuration/#customizing-ryuk-resource-reaper.
I wrote everything in a more detailed report in my https://github.com/project-xavier/xavier-integration/pull/20#issuecomment-515494842 in your PR and it seems like Sergei's fine with this analysis.